config/default/common/config/metadata/layers/airs/RelativeHumidity.md

```diff
@@ -32,15 +32,15 @@ Relative humidity of the air in units of percent at the 700 hPa pressure level,
 Nominal equatorial image resolution 31.25 km/pixel
 
 #### Data Product
-Image initially produced with NRT data. Science quality image replaces NRT when available.  
-Near Real-Time Product: `AIRS2RET_NRT` (AIRS-Only Level 2 Near Real-Time Product)  
-Science Quality Product: `AIRS2RET` (AIRS-Only Level 2 Standard Product)  
-Field name: `RelHum`; pressure level at element 5  
+Image initially produced with NRT data. Science quality image replaces NRT when available.
```

Comment:
Looks like from here down this file didn't get all the `<br>` line breaks that were added elsewhere.

Reply:
🦅 👁️  - Nice catch! I added the <br> and also fixed the URL for the version in the Relative Humidity section.